cffi.asd

```diff
@@ -65,11 +65,28 @@
      (:file "foreign-vars")
      (:file "features")))))
 
-(defmethod operation-done-p ((o test-op) (c (eql (find-system :cffi))))
-  nil)
-
-(defmethod perform ((o test-op) (c (eql (find-system :cffi))))
-  (operate 'asdf:load-op :cffi-tests)
-  (operate 'asdf:test-op :cffi-tests))
+(defsystem :cffi/c2ffi
```

Comment:
ASDF treats '/' specially, the system 'foo/bar' can be put into the 'foo.asd' file and ASDF will still find it.

it's a way to group many related .asd files into one. i propose to convert the rest of the .asd's, too. backwardness can be achieved by retaining a simple cffi-libffi.asd with an empty system that depends on cffi/libffi.